ref/fake-sentry-config-logging

RaduW opened the pull request:
ref(fake-sentry): Fine tune logging in fake sentry.

Make logging more configurable so we don't drown in logs.

tonyo commented:
@RaduW To clarify, when we merge this, what would be the default behaviour? Only warnings/errors?
TBH, I sometimes see some value in seeing uwsgi logs (e.g. to confirm that it gets any requests), what would be a way to reenable those with the new configuration?

RaduW commented:
@tonyo The changes are the following.
- removed the default logging of uwsgi, which was generating one log message for each request
- added a debug message for each request in handler (as a replacement of the above, but with ability to silence it)
- fine tune the logging level with the defaults set as following:
  - messages coming from the fake_sentry logger set to DEBUG, fake_sentry logger is also the logger used the Flask app (so the `app.logger`)
  - messages coming from werkzeug ( general flask infrastructure ) set to WARNING
  - all other loggers captured in the top (`root`) logger set to ERROR

This setup with the configuration in place will log the same  amount of messages as previously but it is easy to adjust ( either by setting the store message to `info` or by setting the app logger to `info`)

With the log configuration removed it will not log each store request (since it sets the app logger to `info` and we log at `debug` level)

**To answer your question:** Setting the app logger (`fake_sentry.fake_sentry`) to `level: DEBUG`, as it is set in the PR should emit a log message for each store request.
